tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -735,7 +731,7 @@ def create_text_embeddings_model(model_path, device, memory_data_collector, **kw
         trust_remote_code=trust_remote_code
     )
     end = time.perf_counter()
-    pooling_type = kwargs.get("emb_pooling_type", "cls")
+    pooling_type = kwargs.get("emb_pooling_type") or "cls"
```

Comment:
```suggestion
    pooling_type = kwargs.get("emb_pooling_type",  "cls")
```

Reply:
kwargs["emb_pooling_type"] can have value and be None.
we can collect kwargs for that function from cmd here https://github.com/openvinotoolkit/openvino.genai/blob/master/tools/llm_bench/llm_bench_utils/model_utils.py#L139 and if emb_pooling_type is not be set in cmd, it will be in kwargs, but its value will be None